command/sub8_mission_control/src/goal_unittest.py

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env python
```

Comment:
See the other unit tests and integration tests for how to do this

[Integration Test](https://github.com/uf-mil/Sub8/tree/master/gnc/sub8_thruster_mapper/test)
[Unittest](https://github.com/uf-mil/Sub8/tree/master/utils/sub8_ros_tools/test_ros_tools)

You should be able to run all tests by doing catkin_make run_tests